flask_server.py

```diff
@@ -59,39 +59,31 @@ def get_system_logs():
     with open(log_file_path, 'r+') as log_file:
         return [log for log in log_file.readlines()]
 
+def get_scraper_status():
+    return {
+            "log_lines": get_log_file_list(),
+            "logged_in": "session_id" in user_settings,
+            "output": scraper_runner.getOutput(),
+            "status": scraper_runner.getStatus()
+        }
+
+def get_scraper_settings():
+    args = scraper_runner.args
+    loop_mode = not scraper_runner.shutting_down and bool(args)
```

Comment:
There's nothing to substitute the bool cast, because this parameter has to be False, not any falsy value like 0, " , etc.. because it's used to query an HTML tag, maybe I can put the check client-side, but I thinks it's like the same.